jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/common/NestedDropdownMenu.java

```diff
@@ -0,0 +1,128 @@
+package com.epam.jdi.light.angular.elements.common;
+
+import com.epam.jdi.light.angular.asserts.NestedDropdownMenuAssert;
+import com.epam.jdi.light.common.ElementArea;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import org.openqa.selenium.By;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.epam.jdi.light.driver.WebDriverFactory.getDriver;
+import static com.epam.jdi.light.elements.init.UIFactory.$$;
+
+/**
+ * Angular slide toggle documentation: https://jdi-docs.github.io/jdi-light/?java#slide-toggle
+ */
+public class NestedDropdownMenu extends UIBaseElement<NestedDropdownMenuAssert> {
+    private String LAST_OPENED_MENU_OPTIONS_XPATH = "(.//div[contains(@class,'mat-menu-after')])[last()]//button";
+    private String LAST_OPENED_MENU_OPTIONS_XPATH_FOR_MENU_WITH_ICONS = "(.//div[contains(@class,'mat-menu-after')])[last()]//button//span";
+
+    protected Button getButton() {
+        return new Button().setCore(Button.class, base());
+    }
+
+    @Override
+    public NestedDropdownMenuAssert is() {
+        return new NestedDropdownMenuAssert().set(this);
+    }
+
+    public void expandMenu() {
```

Comment:
In my opinion there is no need to add postfixes like Menu in all methods

Reply:
Fixed, thanks.